src/components/organisms/NoteDetail/NoteDetail.tsx

```diff
@@ -449,6 +450,36 @@ export default class NoteDetail extends React.Component<
     )
   }
 
+  handlePaste = async (event: ClipboardEvent) => {
+    event.preventDefault()
+
+    const { storage, addAttachments } = this.props
+
+    const files = getFilesFromClipboard(event).filter((file) =>
+      file.type.startsWith('image/')
+    )
+
+    const attachments = await addAttachments(storage.id, files)
+    if (attachments.length === 0) return
+
+    this.setState(
```

Comment:
This will paste the image at the end of the note which can be annoying if the user want to paste an image at the cursor position.

Reply:
I'll change that !